lib/index.js

```diff
@@ -29,42 +48,65 @@ var Iterable = module.exports = integration('Iterable')
  * @api public
  */
 
-Iterable.prototype.track = function(payload, fn){
+Iterable.prototype.track = function(track, fn) {
   return this
     .post('/events/track')
     .type('json')
     .set(headers(this.settings))
-    .send(payload)
+    .send(track)
     .end(this.handle(fn));
+}
+
+/**
+ * Added/Removed Product.
+ *
+ * Regular track call, unless properties.products is present
+ *
+ * @param {Track} track
+ * @param {Function} fn
+ * @api public
+ */
+
+Iterable.prototype.addedProduct =
+Iterable.prototype.removedProduct = function(track, fn) {
+  if (!track.products().length) {
```

Comment:
What happens if ```products``` is an empty array? We want it to go to ```updateCart``` not ```track```; this can happen if you remove the last product in your cart. will ```!0``` return true or false in JS?